src/edge.hal-interface/src/hal_interface.py

```diff
@@ -355,20 +355,20 @@ def get_signal_id(self):
         return '{}.{}'.format(self.frame_id, self.signal)
 
 class MockBusObserver(BusObserver):
-    def __init__(self, bus_config, hal_broker):
-        super(MockBusObserver, self).__init__(bus_config, hal_broker)
+    def __init__(self, bus_config, protocol_gateway):
+        super(MockBusObserver, self).__init__(bus_config, protocol_gateway)
 
         # pylint: disable=anomalous-backslash-in-string
-        self.topic_regex = re.compile('^{}([^\.]+)\.(.+)\/(?:start|stop)$'.format((self.hal_broker.topic_ns or '').replace('\\/', '\\\\/')))
+        self.topic_regex = re.compile('([^\.]+)\.(.+)\/(?:start|stop)$')
```

Comment:
Should the topic namespace not be considered anymore?

Reply:
The callbacks are called with the namespace stripped off. It is not passed into subscribe and publish methods, so not got in the callbacks.